Orange/canvas/report/report.py

```diff
@@ -232,10 +232,10 @@ def fmtnum(s):
                 return s
 
         def report_list(data,
-                        header_rows=header_rows, header_columns=header_columns):
+                        header_rows=header_rows, header_columns=header_columns, confusion_rows=confusion_rows):
             cells = ["<td>{}</td>", "<th>{}</th>"]
             return join("  <tr>\n    {}</tr>\n".format(
-                join(cells[rowi < header_rows or coli < header_columns]
+                join(cells[rowi < header_rows or coli < header_columns or rowi == 3 + confusion_rows]
```

Comment:
1 row = empty
2 row = class names
3 = class names
So we have to add the confusion_rows  + 3.
Hope I am clear .